javascripts/background_scripts/posting_process.js

```diff
@@ -82,6 +82,7 @@ var postingProcess = {
     } else {
       var notification = new Notification("There is already a pending post",
         {icon: "images/logo_48.png"});
+      notification.show();
```

Comment:
It looks like `notification.show()` was incorrectly removed in db085826a54704e642e637f589d8703d1cd6e28b.  Good job adding it back in.